packages/mds-ingest-service/repository/index.ts

```diff
@@ -482,6 +488,23 @@ export const IngestRepository = ReadWriteRepository.Create(
         }
       },
 
+      createTelemetryAnnotations: async (telemetryAnnotations: TelemetryAnnotationDomainCreateModel[]) => {
```

Comment:
Will these be written in small or large batches? If large, it might be worth using the chunked insert approach.

Reply:
I like your `Repository.asChunksForInsert()` helper! 
This is going to be populated via an annotation processor, I'm assuming those message-sizes will be chunked the same way that the original Telemetry batches from providers are chunked? 
It might be moot, depending where this `createTelemetryAnnotations` gets called from.

Reply:
Good question @drtyh2o. It'll be one at a time, in the telemetry processor.